docs/guides/admin/docs/configuration/https/apache-httpd.md

```diff
@@ -0,0 +1,79 @@
+Enable HTTPS using Apache httpd
+===============================
+
+> [Using Nginx as reverse proxy for Opencast](nginx.md) is the preferred way of running Opencast.
+> Refer to the [Nginx guide for configuration instructions](nginx.md) for that type of setup.
+
+This guide will help you to configure httpd to act as HTTP(S) proxy for Opencast.
+
+
+Opencast Configuration
+----------------------
+
+Make sure to use `https` as protocol for `org.opencastproject.server.url` in `etc/custom.properties`.
+
+```ini
+org.opencastproject.server.url=https://example.opencast.org
+```
+
+Also enable secure cookies in `etc/org.ops4j.pax.web.cfg`:
+
+```ini
+org.ops4j.pax.web.session.cookie.secure=true
+org.ops4j.pax.web.session.cookie.httpOnly=true
+```
```

Comment:
I think these lines should also go into nginx-documentation.

Reply:
No. You don't need that since Nginx takes care of that :)
That's also, what I think this would do for Apache httpd and I tend to replace this. 
But I have no test system to try it out any longer:

```
  Header edit Set-Cookie ^(.*)$ $1; HttpOnly; Secure; SameSite=None
```

Reply:
This does the same as the nginx code. For consistency reasons, I would prefer this option instead of the ini file.